### PR TITLE
Adh update deps

### PR DESCRIPTION
**Pull request by phoenixy1**
- Fix deps to make security team happy
- Update README to clarify how to get usable accounts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ Would you like to be the first to own a pre-owned hoverboard? Well, you're in lu
 
 When you first start up the app, you'll be prompted to connect to a bank to load up your liabilities, and connect to your Payroll provider (or bank) to load up your sources of income! We provide you with the opportunity to add additional sources of income as well.
 
-When asked to connect to a bank while in Sandbox mode, supply the credentials **`user_bank_income`** and **`{}`** and the MFA code of `1234` if you're asked for it. This is different from the credentials you will be shown at the bottom of the page! While the `user_good` / `pass_good` credentials will work, they won't have much in the way of income streams associated with them. For more test accounts that work great with Income, see the [docs](https://plaid.com/docs/sandbox/test-credentials/#credit-and-income-testing-credentials).
+>[!IMPORTANT]
+> When adding **liabilities** in the Sandbox environment, you must use a non-OAuth bank (like Houndstooth Bank) and select a liabilities-compatible account, like a credit card. Selecting an OAuth bank in the Sandbox or picking an account that doesn't work with liabilities will cause link to fail with a "no supported liabilities accounts" error.
```
Comment (ToddKerpelman) [inline README.md:33]:
```suggestion
> When adding **liabilities** in the Sandbox environment, you must use a non-OAuth bank (like Houndstooth Bank) and select a liabilities-compatible account, like a credit card. Selecting an OAuth bank in Sandbox or picking an account that doesn't work with liabilities will cause link to fail with a "no supported liabilities accounts" error.
```

**Review by ToddKerpelman — COMMENTED** [review]
LGTM but I did that thing where I made a small suggestion and I'll need to re-approve.

**Review by ToddKerpelman — APPROVED** [review]

**Comment by phoenixy1** [comment]
> LGTM but I did that thing where I made a small suggestion and I'll need to re-approve.

turns out this repo isn't set up that way! :-D

**Review by atim124 — COMMENTED** [review]
Yyyyyyyy

**Pull request merged**